interface/modules/custom_modules/oe-module-claimrev-connect/public/claims.php

```diff
@@ -0,0 +1,334 @@
+<?php   
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="claims";
+    require_once "../../../../globals.php";
+   
+    require_once '../src/ClaimSearchModel.php';
+    require_once '../src/ClaimSearch.php';
+    require_once '../src/ClaimRevApi.php';
+    require_once '../src/UploadEdiFileContentModel.php';
+
+   
+    use OpenEMR\Modules\ClaimRevConnector\ClaimSearch;
+    use OpenEMR\Modules\ClaimRevConnector\ClaimSearchModel;
+
+    class ClaimsPage
+    {
+        public static function SearchClaims($postData)
+        {
+            $firstName = $_POST['patFirstName']; 
+            $lastName = $_POST['patLastName']; 
+            $startDate = $_POST['startDate']; 
+            $endDate = $_POST['endDate']; 
+
+            $model = new ClaimSearchModel();
+            $model->patientFirstName = $firstName;
+            $model->patientLastName = $lastName;
+            $model->receivedDateStart = $startDate;
+            $model->receivedDateEnd = $endDate;
+
+            $data = ClaimSearch::Search($model);
+            return $data;
+        }
+    }
+?>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+    <title>ClaimRev Connect - Claims</title>
```

Comment:
Translate strings. Basically 2 functions that do this depending on if the string is inside html attribute (xla()) or not inside html attribute (xlt()). Couple rules for translations are no leading/trailing spaces and no colons at ned of translated strings. In this case would be:
```
<title><?php echo xlt(ClaimRev Connect - Claims); ?></title>
```

Reply:
complete